app/src/components/market/sections/market_buy/scalar_market_buy.tsx

```diff
@@ -344,7 +344,11 @@ export const ScalarMarketBuy = (props: Props) => {
       setDisplayAmountToFund(new BigNumber('0'))
       setAmount(new BigNumber(0))
       setStatus(Status.Ready)
-      setMessage(`Successfully bought ${formatNumber(sharesAmount)} ${balances[outcomeIndex].outcomeName} shares.`)
+      setMessage(
+        `Successfully bought ${formatNumber(sharesAmount, baseCollateral.decimals)} ${
```

Comment:
We don't want this decimal value here, probably just want the default parameter of two decimals